### ISSUE TITLE
Kvaser CAN Interface - Error opening reader: -1 - Initialization of the CAN interface failed.

### ISSUE THREAD

**Issue by Durant35** [astuff/kvaser_interface#43]
```bash
...
process[rosout-1]: started with pid [4343]
started core service [/rosout]
process[kvaser_can_bridge-2]: started with pid [4346]
[ INFO] [1561980220.508395061]: Kvaser CAN Interface - Got hardware_id: 40496
[ INFO] [1561980220.509427126]: Kvaser CAN Interface - Got can_circuit_id: 0
[ INFO] [1561980220.510545732]: Kvaser CAN Interface - Got bit_rate: 500000
[ERROR] [1561980220.585378959]: Kvaser CAN Interface - Error opening reader: -1 - Initialization of the CAN interface failed.
[kvaser_can_bridge-2] process has died [pid 4346, exit code 255, cmd /home/sensetime/catkin_ws/devel/lib/kvaser_interface/kvaser_can_bridge __name:=kvaser_can_bridge __log:=/home/sensetime/.ros/log/ac49e8d4-9bf2-11e9-bfe9-a0369f51a08c/kvaser_can_bridge-2.log].
log file: /home/sensetime/.ros/log/ac49e8d4-9bf2-11e9-bfe9-a0369f51a08c/kvaser_can_bridge-2*.log
...
```

**Comment by Durant35** [comment]
```bash
$ ./listChannels 
Canlib version 8.27
Found 1 channel(s).
channel  0 = Kvaser Leaf Light v2,	73-30130-00685-0, 40496, 4.1.0.844
```

**Comment by JWhitleyAStuff** [comment]
@Durant35 Here are some questions to help you:

- Have you restarted your machine after installing the Kvaser SDK?
- Which version of the SDK did you download?
- How did you install the SDK?
- How did you install `kvaser_interface`?

**Comment by Durant35** [comment]
@JWhitleyAStuff  Seems that it's the "too high" version of the Kvaser linuxcan SDK installed from PPA following [Fix linuxcan (Kvaser SDK) Install](https://autonomoustuff.atlassian.net/wiki/spaces/RW/pages/17473096/Fix+linuxcan+Kvaser+SDK+Install), which is `8.27` in my **Ubuntu16.04 docker with ros kinetic**
```bash
$ uname -a
Linux gary-MS-7A71 4.4.0-130-generic #156-Ubuntu SMP Thu Jun 14 08:53:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

And **kvaser_interface** worked after remove all `libcanlib.so.*` and build&install from _linuxcan.tar.gz_, and
```bash
$ ./listChannels 
Canlib version 5.20
Found 1 channel(s).
channel  0 = Kvaser Leaf Light v2,	73-30130-00685-0, 40496, 4.1.0.844
```

But, everything has a but...**delphi_esr** from **as_drivers** launched, but no data echoed in any topic, but **kvaser_interface** has in the same docker

**Comment by Durant35** [comment]
@JWhitleyAStuff Fixed for **kvaser_interface**, thus close this issue. Still hope to hear from  you about the situation as mentioned above, thanks a lot.

**Comment by JWhitleyAStuff** [comment]
@Durant35 Which version of linuxcan.tar.gz did you download from Kvaser? You can get any version back to the beginning from their website. Also, 8.27 is not a valid Canlib version. Are you talking about the version of the package you downloaded from the PPA? The latest published version in my PPA is 5.27.776 for Ubuntu 16.04. On their website, you can now download 5.28.846 (I'll update my PPA soon).

**Comment by Durant35** [comment]
That’s so strange... As you can see above, the version I mentioned is the info from
**listChannels**
